code/_onclick/hud/screen_objects.dm

```diff
@@ -23,6 +23,8 @@
 	master = null
 	return ..()
 
+/obj/screen/orbit(atom/A, radius, clockwise, rotation_speed, rotation_segments, pre_rotation, lockinorbit)

```

Comment:
I'd rather have signatures same in case some crazy person decides to call them through named args.